### PR TITLE
Add Mathematica evaluate module

### PR DESCRIPTION
**Pull request by Tyilo**

### CODE REVIEW

File: mathematica_eval.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+__dependencies__ = ['mathematica']
+
+if not which('wolframscript'):
+    raise Exception("`wolframscript` is not in $PATH.")
+
+ICON_PATH = iconLookup('wolfram-mathematica')
+
+def handleQuery(query):
+    if not query.isTriggered:
+        return
```
Comment (ManuelSchneid3r) [inline mathematica_eval.py]: Thats not allowed atm but with the next version ;D

File: mathematica_eval.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        success = True
+    else:
+        item.text = ''
+        item.subtext = 'Type a Mathematica expression'
+        success = False
+
+    if success:
+        item.addAction(ClipAction('Copy result to clipboard', result))
+
+    return [item]
```
Comment (ManuelSchneid3r) [inline mathematica_eval.py]: 0.14.12 will allow returning single items

**Comment by Tyilo** [comment]
It now returns the item directly. Ready to be merged?

**Comment by Tyilo** [comment]
Ping @ManuelSchneid3r

File: mathematica_eval.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        result = str(output.strip(), 'utf-8')
+        item.text = result
+        item.subtext = 'Result'
+        success = True
+    else:
+        item.text = ''
+        item.subtext = 'Type a Mathematica expression'
+        success = False
+
+    if success:
```
Comment (ManuelSchneid3r) [inline mathematica_eval.py]: why dont you just move this to line 35 and remove *success* variable?

Comment (Tyilo) [reply mathematica_eval.py]: I think I just used a template for some other python extension, but I have removed it now :)

**Comment by ManuelSchneid3r** [comment]
Hi @tyilo do you plan to port your extensions to the new api or can I remove them from the `.archive`?

**Comment by tyilo** [comment]
@ManuelSchneid3r Ported: https://github.com/albertlauncher/python/pull/174

**Pull request merged**